src/index.ts

```diff
@@ -167,26 +194,35 @@ export function HTPipe<
 // two parameters with possibly added inputs
 export function HTPipe<
   TLeft extends ClashlessInitPreContext<TLeft, TRight> &
```

Comment:
- [ ] you forgot to update the "automatic type guessing" overload above.

Reply:
done

Reply:
wait no... it's still missing stuff...  look at the type parameters
- [ ] more automatic type guessing

Reply:
done